testindex/validators.py

```diff
@@ -240,9 +240,7 @@ def run(self):
                 depends_on = entry["depends-on"]
                 if not isinstance(depends_on, list):
                     depends_on = [depends_on]
-                matcher = re.compile("^(([0-9a-zA-Z_-]+[.]{1})*([0-9a-zA-Z_-]+)\
-                                     {1}[/]{1})?[0-9a-zA-Z_-]+[/]{1} \
-                                     [0-9a-zA-Z_-]+[:]{1}([0-9a-zA-Z_-]+\.?)+$"
+                matcher = re.compile("^(([0-9a-zA-Z_-]+[.]{1})*([0-9a-zA-Z_-]+){1}[/]{1})?[0-9a-zA-Z_-]+[/]{1}[0-9a-zA-Z_-]+[:]{1}([0-9a-zA-Z_-]+\.?)+$"
```

Comment:
I would suggest, we add some tests with different samples to be tested with regex.